app/build.gradle

```diff
@@ -171,6 +171,9 @@ dependencies {
 
 // make crashlytics upload task dependent on the assemble task for all build variants
 afterEvaluate {
+
+    crashlyticsUploadDistributionProductionDebug.dependsOn assembleProductionDebug
+
```

Comment:
this code should be removed, not uncommented. it was replaced by the `android.applicationVariants.all` code right below it